client/src/common/components/Members/components/MembersForm.jsx

```diff
@@ -65,48 +86,86 @@ class MembersForm extends PureComponent {
 
   handleBlur = () => this.setState({ isFocused: false });
 
+  handleClickOutside = event => {
```

Comment:
What happens when you click outside during sending invitations? Does "cannot setState on unmounted component..." error appears or something?

Reply:
Everything works as expected. No errors